popup.js

```diff
@@ -0,0 +1,41 @@
+const close1 = document.querySelector('.close1');
+const close2 = document.querySelector('.close2');
+const close3 = document.querySelector('.close3');
+const close4 = document.querySelector('.close4');
+
+// card 1
+document.querySelector('.see-pro1').addEventListener('click', () => {
+  document.querySelector('.pop-1').classList.toggle('active');
+  // document.querySelector('.overlay').classList.toggle('active');
+});
+
+close1.addEventListener('click', () => {
+  document.querySelector('.pop-1').classList.toggle('active');
+});
+
+// card 2
+document.querySelector('.see-pro2').addEventListener('click', () => {
+  document.querySelector('.pop-2').classList.toggle('active');
+});
+
+close2.addEventListener('click', () => {
+  document.querySelector('.pop-2').classList.toggle('active');
+});
+
+// card 3
+document.querySelector('.see-pro3').addEventListener('click', () => {
+  document.querySelector('.pop-3').classList.toggle('active');
+});
+
+close3.addEventListener('click', () => {
+  document.querySelector('.pop-3').classList.toggle('active');
+});
+
+// card 4
+document.querySelector('.see-pro4').addEventListener('click', () => {
+  document.querySelector('.pop-4').classList.toggle('active');
+});
+
```

Comment:
- [ ] Good job for implementing the popup same as Figma design but the only comment here is for the responsiveness try to make your pop-up responsive until 320px 
also, try to add padding or margin to this button in the desktop version so it looks better 
1- 

![image](https://user-images.githubusercontent.com/74744787/181833941-57ebc33b-3481-459c-9d4d-fae6696df77b.png)

2- 
![image](https://user-images.githubusercontent.com/74744787/181834018-3047371a-3176-4cfc-b61f-3ebb1479368c.png)